app/controllers/api/public/v1/providers/locations_controller.rb

```diff
@@ -3,8 +3,13 @@ module Public
     module V1
       module Providers
         class LocationsController < API::Public::V1::ApplicationController
+          PERMITTED_INCLUSIONS = %w[recruitment_cycle provider course].freeze
```

Comment:
i don't think we need this. we did have this on a pull request but it didn't get merge. the serialiser already whitelists this as you have to specify it in `belongs_to`. therefore i think we can delete this line and change `include_param` to `params.fetch(:include, "")`

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1523/commits/9bf86e182e96b73d94ee29d23ace4619f2596af7